src/clj_s3_client/core.clj

```diff
@@ -3,12 +3,23 @@
             [camel-snake-kebab.extras :refer [transform-keys]]
             [clojure.set :refer [difference]])
   (:import [com.amazonaws.services.s3 AmazonS3Client]
-           [com.amazonaws.services.s3.model AmazonS3Exception ObjectMetadata]
+           [com.amazonaws.services.s3.model CannedAccessControlList PutObjectRequest AmazonS3Exception ObjectMetadata]
            [com.amazonaws ClientConfiguration]
            [com.amazonaws.regions Regions]
            [java.io InputStream]
            [java.nio ByteBuffer]))
 
+(def access-control-list-mapping
```

Comment:
Or you can just name the map as `acl->canned-access-control-list`?

map acts as a function too.